sdcm/nemesis.py

```diff
@@ -4034,7 +4034,7 @@ def _add_new_node_in_new_dc(self) -> BaseNode:
             scylla_yml.rpc_address = new_node.ip_address
             scylla_yml.seed_provider = [SeedProvider(class_name='org.apache.cassandra.locator.SimpleSeedProvider',
                                                      parameters=[{"seeds": self.tester.db_cluster.seed_nodes_ips}])]
-            if self.cluster.params.get("endpoint_snitch").endswith("GossipingPropertyFileSnitch"):
+            if self.cluster.params.get("endpoint_snitch", "").endswith("GossipingPropertyFileSnitch"):
```

Comment:
@soyacz we need to fix this, `SctConfiguration.get` doesn't have default argument

Reply:
fixed